src/PointwiseFunctions/Hydro/TransportVelocity.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace hydro {
+
+namespace Tags {
+/// \brief Tag containing the transport velocity.
+///
+/// The transport velocity is defined as \f$v_t^i=\alpha v^i-\beta^i\f$,
+/// with $v^i$ being the spatial velocity, $\alpha$ the lapse, and
+/// $\beta^i$ the shift.
+template <typename DataType, size_t Dim, typename Fr = Frame::Inertial>
+struct TransportVelocity : db::SimpleTag {
```

Comment:
I would park the simple tag (for better or worse) in `Hydro/Tags.hpp`. We are going to do a big refactor that changes things to `Hydro/Tags/TransportVelocity.hpp`, but that's for the future.